src/instantiate.jl

```diff
@@ -69,6 +131,18 @@ function newpage(; path="page", overwrite=false)
             chmod(joinpath(root, file), 0o644)
         end
     end
+    # Try filling `config.md`
+    name = "config.md"
+    src = joinpath(store, name)
+    config = read(src, String)
+    dst = joinpath(path, name)
+    try 
+        config = replace(config, r"{{ ([a-z_]+) }}" => x->fill(Val(Symbol(x[4:end-3]))))
```

Comment:
1. I don't think this should be in the try (will not fail)
2. a bit more robust and possibly more readable

```julia
# outside
const INFER_PAT = r"{{\s*([a-z_]+)\s*}}"
# inside
config = replace(config, INFER_PAT => 
                 s -> match(INFER_PAT, s).captures[1] |> Symbol |> Val |> fill)
```

Reply:
Note that it would be even more readable to just do `|> Symbol |> fill` and have a `fill` that takes a Symbol, passes it to a val then calls fill again:

```julia
fill(w::Symbol) = fill(Val(w))
```

Reply:
Ah yeah I didn't get the first match in replace and didn't think further to just match it again :smile:

Reply:
Implemented but maybe the discourse post gives us something better